test/unit/src/signals/SignalsTest.cpp

```diff
@@ -102,8 +102,9 @@ TEST_CASE( "signals/Signals" )
 	{
 		class MemberClass {
 		public:
+			MemberClass() { mAccum = 0; }
 			void handler()	{ mAccum++;	}
-			int mAccum = 0;
```

Comment:
My assumption was that it had something to do with [this](http://connect.microsoft.com/VisualStudio/feedbackdetail/view/896703/a-fatal-error-c1506-unrecoverable-block-scoping-error-in-c-compiler-is-reported-for-class-with-default-member-values-definitions-inside-a-nested-block), as I this is the error I was getting at compile time. if i moved the class def outside of the SECTION, it works just fine also.